### PR TITLE
feat(server): add internal/experimental cmd for getting action statuses

### PR DESCRIPTION
**Pull request by eysi09**
**What this PR does / why we need it**:

This is an internal/experimental cmd that'll only be used by the web dashboard for now.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: core/src/server/commands.ts [hunk 1]
```diff
@@ -250,7 +256,42 @@ export class _GetDeployStatusCommand extends ConsoleCommand {
...
+  streamEvents = false
+
+  outputsSchema = () => joi.object()
+
+  async action({ garden, log }: CommandParams): Promise<CommandResult<GetActionStatusesCommandResult>> {
+
+    const router = await garden.getActionRouter()
+    const graph = await garden.getResolvedConfigGraph({ log, emit: true })
+
+    const actions = await Bluebird.props({
```
Comment (thsig) [inline core/src/server/commands.ts:286]: Maybe `statuses` would be a better name here?

File: core/src/actions/helpers.ts [hunk 2]
```diff
@@ -140,3 +146,90 @@ export function getRelativeActionConfigPath(projectRoot: string, action: Action)
   const relPath = relative(projectRoot, action.configPath() ?? "")
   return relPath.startsWith("..") ? relPath : "." + sep + relPath
 }
+
+export async function getDeployStatuses(router: ActionRouter, graph: ResolvedConfigGraph, log: Log) {
```
Comment (thsig) [inline core/src/actions/helpers.ts:150]: How about renaming these to `getDeployStatusPayloads`/`getBuildStatusPayloads` etc. to more clearly distinguish this from the other `getDeployStatuses` function?

**Review by thsig — APPROVED** [review]
Looks good! Had a couple of non-blocking comments there, I'll leave it up to you whether to accept/address them or not.

File: core/src/server/commands.ts (hunk 1 above)
Comment (eysi09) [reply core/src/server/commands.ts:286]: It's an action map though that we're returning. And it jives with the result interface.

**Pull request merged**